doc/source/_toc.yml

```diff
@@ -322,6 +320,8 @@ parts:
                 sections:
                   - file: cluster/cluster_under_construction/ray-clusters-on-vms/examples/setting-up-ray-cluster-for-production/todo
             - file: cluster/cluster_under_construction/ray-clusters-on-vms/references/index
+          - file: cluster/cluster_under_construction/running-applications-on-ray-clusters/index
+            title: Running Applications on Ray Clusters
```

Comment:
Related to above, maybe "Run an Application on a Ray Cluster" instead (or change the other two to match this).

Reply:
Hm, I think "Deploying" and "Running" are better. Will change them to be consistent